job_controller/main.py

```diff
@@ -22,14 +23,18 @@ class JobController:
     """
 
     def __init__(self) -> None:
+        db_ip = os.environ.get("DB_IP", "")
```

Comment:
Is the fallback value useful here?

Reply:
Hmmm, then not sure why I gave a default here, I'll fix it on main.